pages/design/test-cases.md

```diff
@@ -0,0 +1,68 @@
+# ACT Test Case Design
+
+The goal of test cases in ACT rules is allow implementors to varify that their implementation is sufficiently consistent with the ACT rule. This requires that a rule has test cases for all of the "important" parts of it. Beyond that, it is up to the rule authors to determine how extensive to make the list of test cases. A list of test cases full of edge cases is going to ensure greater consistency between implementors, but may result in fewer implementors deciding to adopt the rule.
+
+To strike a good balance in test cases, the following principles should be considered:
+
+1. Test every "condition" in the rule
+1. Test one thing at a time
+1. Ensure consistency with accessibility 
+1. Tests should be small
```

Comment:
```suggestion
1. Keep test cases small
```

Suggestion to keep the grammar perspective consistent between the bullet points rather than to change the meaning, so do adjust if the meaning is changed.